include/fuzzuf/algorithms/nezha/create.hpp

```diff
@@ -69,13 +69,14 @@ auto CreateRunSingleTarget(const fs::path &target_path,
   const auto output_file_path = create_info.output_dir / "result";
   const auto path_to_write_seed = create_info.output_dir / "cur_input";
 
-  std::unique_ptr<NativeLinuxExecutor> executor(new NativeLinuxExecutor(
+  std::shared_ptr<NativeLinuxExecutor> nle(new NativeLinuxExecutor(
       {target_path.string(), output_file_path.string()},
       create_info.exec_timelimit_ms, create_info.exec_memlimit,
       create_info.forksrv, path_to_write_seed, create_info.afl_shm_size,
       create_info.bb_shm_size, true));
+  auto executor = std::make_unique<DynLibFuzzerCapable>(std::move(nle));
```

Comment:
If I'm not mistaken, you need `DynLibFuzzerCapable::DynLibFuzzerCapable(T&&)` to make `std::move(nle)` actually move `nle`. Probably `T&&` would be casted to `const T&` because we don't define move constructors.

Reply:
I added move constructor in [5be878d](https://github.com/fuzzuf/fuzzuf/pull/41/commits/5be878d42a3206524a6732f7a10143dd4bb1abc8) and
[bdacfde](https://github.com/fuzzuf/fuzzuf/pull/41/commits/bdacfde1fd6e315304ac1a44d86819b109ac7971). I'm not sure the classes accept copy constructors. Should we remove them?